include/dxc/Support/HLSLOptions.td

```diff
@@ -265,6 +265,7 @@ def Gfp : Flag<["-", "/"], "Gfp">, HelpText<"Prefer flow control constructs">, F
 // /Gdp - disable effect performance mode - unsupported
 def Ges : Flag<["-", "/"], "Ges">, HelpText<"Enable strict mode">, Flags<[CoreOption]>, Group<hlslcomp_Group>;
 def Gis : Flag<["-", "/"], "Gis">, HelpText<"Force IEEE strictness">, Flags<[CoreOption]>, Group<hlslcomp_Group>;
+def NoLineDirectives : Flag<["-"], "NoLineDirectives">, HelpText<"Not use line directives">, Flags<[CoreOption]>, Group<hlslcomp_Group>;
```

Comment:
We should call the flag `IgnoreLineDirectives` because it controls the behavior when reading source not when producing it.